processing.py

```diff
@@ -315,8 +315,16 @@ def process_batch_all(timestamp, extrinsicEvents, ex_dict):
 
 def get_timestamp(result) -> str:
     res = result["block"]["extrinsics"]
-    ts = res[0].value["call"]["call_args"][0]["value"]
-    return ts
+    s = res[0].value["call"]["call_args"][0]["value"]
+    timestamp = ""
+    if isinstance(s, int):
+        timestamp = s
+    else:
+        tms = s.split(".")
+        ts = tms[0]
+        ms = int(tms[1]) / 1000 if len(tms) > 1 else 0
+        timestamp = int(datetime.strptime(ts, "%Y-%m-%dT%H:%M:%S").timestamp()) * 1000 + ms
```

Comment:
timestamp = int(datetime.strptime(ts, "%Y-%m-%dT%H:%M:%S").timestamp())
```suggestion
        timestamp = int(datetime.strptime(ts, "%Y-%m-%dT%H:%M:%S.%f").timestamp())
```

Reply:
https://docs.python.org/3/library/datetime.html#strftime-and-strptime-format-codes

Reply:
time could be without microseconds